src/lancie-admin-progress/lancie-admin-progress-meter.html

```diff
@@ -0,0 +1,45 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-progress/paper-progress.html">
+
+<dom-module id="lancie-admin-progress-meter">
```

Comment:
Is a new element needed?

Reply:
I wasn’t sure what I was aiming for, but if we want to expand it, we can. I can just change it and put it in the parent. What does the rest think about this?